pygfx/renderers/wgpu/wgsl/light_pbr.wgsl

```diff
@@ -100,83 +100,42 @@ fn computeMultiscattering(normal: vec3<f32>, view_dir: vec3<f32>, specular_color
 }
 
 fn RE_IndirectSpecular_Physical(radiance: vec3<f32>, irradiance: vec3<f32>,
-        geometry: GeometricContext, material: PhysicalMaterial, reflected_light: ReflectedLight) -> ReflectedLight{
+        geometry: GeometricContext, material: PhysicalMaterial, reflected_light: ptr<function, ReflectedLight>){
```

Comment:
I had to look up the pointer type definition syntax to understand this, I thought it was some kind of to-from mapping (function that returns a type). But instead, `function` means the data is stored in the function's address space. Is that correct?

Reply:
Yes, it is a local variable, using memory locations in the "function" address space.

See:  https://www.w3.org/TR/WGSL/#ref-ptr-types